ipaserver/plugins/host.py

```diff
@@ -691,7 +691,8 @@ def pre_callback(self, ldap, dn, entry_attrs, attrs_list, *keys, **options):
             # save the password so it can be displayed in post_callback
             setattr(context, 'randompassword', entry_attrs['userpassword'])
 
-        entry_attrs['usercertificate'] = options.get('usercertificate', [])
+        entry_attrs['usercertificate'] = list(
+            options.get('usercertificate', []))
```

Comment:
unless I'm mistaken, this line is not needed at all. The entry_attrs already contains the usercertificate attribute as a list (because the class inherits from LDAPCreate).